worlds/openrct2/Options.py

```diff
@@ -0,0 +1,932 @@
+    """Includes an ATM in the item list, regardless of whether it normally appears in the chosen scenario."""
+    display_name = "Include ATM"
+
+class IncludeFirstAid(OpenRCT2OnToggle):
+    """Includes a First Aid Room in the item list, regardless of whether it normally appears in the chosen scenario."""
+    display_name = "Include First Aid Room"
+
+class AllRidesAndSceneryBase(OpenRCT2Toggle):
+    """Adds every ride and scenery pack in the base game to the item pool. This will make for a significantly longer scenario."""
+    display_name = "Include All Rides and Scenery (Base Game)"
+
+class AllRidesAndSceneryExpansion(OpenRCT2Toggle):
+    """Adds every ride and scenery pack in the RCT2 Expansion Packs to the item pool. If this is true the base game rides and 
+    scenery will also be included. This will make for a significantly longer scenario."""
+    display_name = "Include All Rides and Scenery (Expansion Packs)"
+
+class Skips(Range):
+    """By default, every game starts with a single skip to ignore a shop requirement. This will add additional skips to 
+    be found in the item pool."""
+    display_name = "Skips"
+    range_start = 0
+    range_end = 100
+    default = 7
+
+openrct2_option_groups = [
+    OptionGroup("Scenario Options", [
+        SelectedScenario,
+        SelectedDifficulty,
+        SelectedScenarioLength,
+        SelectedRandomizationRange,
+        RandomizeParkValues,
+        SelectedIntensity,
+        SelectedStatReRolls,
+    ]),
+    OptionGroup("Goal Options", [
+        GuestObjective,
+        ParkValueObjective,
+        RollerCoasterObjective,
+        RollerCoasterExcitement,
+        RollerCoasterIntensity,
+        RollerCoasterNausea,
+        RequiredUniqueRides,
+        LocalityOfUniqueRides,
+        ParkRatingObjective,
+        PayOffLoan,
+        MonopolyMode,
+        Fireworks
+    ]),
+    OptionGroup("Rules", [
+        SelectedDifficultGuestGeneration,
+        SelectedDifficultParkRating,
+        SelectedForbidHighConstruction,
+        SelectedForbidLandscapeChanges,
+        SelectedForbidMarketingCampaigns,
+        SelectedForbidTreeRemoval
+    ]),
+    OptionGroup("Shop Options", [
+        ShopMinimumExcitement,
+        ShopMaximumExcitement,
+        ShopMinimumIntensity,
+        ShopMaximumIntensity,
+        ShopMinimumNausea,
+        ShopMaximumNausea,
+        ShopMinimumLength,
+        ShopMaximumLength,
+        ShopMinimumTotalCustomers,
+        ShopMaximumTotalCustomers,
+        BalanceGuestCounts,
+        SelectedVisibility
+    ]),
+    OptionGroup("Item & Trap Options", [
+        Filler,
+        IncludeATM,
+        IncludeFirstAid,
+        IncludeGamespeedItems,
+        Skips,
+        FurryConventionTraps,
+        BathroomTraps,
+        SpamTraps,
+        LoanSharkTraps,
+        AllRidesAndSceneryBase,
+        AllRidesAndSceneryExpansion
+    ]),
+]
+
+@dataclass
+class openRCT2Options(PerGameCommonOptions):
```

Comment:
Now your wondering what is a mixin, well its what you put here 
`class openRCT2Options(PerGameCommonOptions, DeathLinkMixin):`
you can import `DeathLinkMixin` at the top from Options like everything else